spec/requests/posts_spec.rb

```diff
@@ -3,15 +3,30 @@
 RSpec.describe 'Posts', type: :request do
   describe 'GET /index' do
     it 'returns http success' do
-      get '/posts/index'
+      get '/users/:user_id/posts/'
       expect(response).to have_http_status(:success)
     end
+
+    it 'returns http success' do
+      get '/users/:user_id/posts/:id'
+      expect(response).to have_http_status(:success)
+    end
+
+    it 'should include correct placeholder' do
+      get '/users/:user_id/posts/:id'
+      expect(response.body).to include('Here are the posts for user')
+    end
```

Comment:
- [ ] These two tests do not belong under the `GET /index` describe block, kindly remove them as they are show action tests.

- [ ] Kindly include a test to ensure the correct template is being rendered:

  ```
  expect(response).to render_template(:index)
  ```